lib/src/global_packages.dart

```diff
@@ -474,14 +475,44 @@ Try reactivating the package.
         result.packages.removeWhere((id) => id.name == 'pub global activate');
 
         final newLockFile = await result.downloadCachedPackages(cache);
+        final report = SolveReport(
+          SolveType.get,
+          entrypoint.workspaceRoot.dir,
+          entrypoint.workspaceRoot.pubspec,
+          entrypoint.workspaceRoot.allOverridesInWorkspace,
+          entrypoint.lockFile,
+          newLockFile,
+          result.availableVersions,
+          cache,
+          dryRun: true,
+          enforceLockfile: true,
+          quiet: false,
+        );
+        await report.show(summary: true);
+
         final sameVersions = entrypoint.lockFile.samePackageIds(newLockFile);
+
```

Comment:
Please add a comment here explaining why we do this thing with creating a lockfile, and comparing the lcokfile, etc.

A comment that says: We need to do this because globally activated packages can have dependencies on SDK-packages, bla bla bla...

Because next time we look at this code, we won't remember why it is we do this resolution stuff here. When the method is recompile.

Reply:
Done